client/src/views/Shorten.js

```diff
@@ -73,20 +78,27 @@ const Shorten = () => {
                     </div>
                     <h2>Autogenererad förkortad länk</h2>
                     <p>Slumpa en fyra karaktärer lång sträng.</p>
-                    <form className="row">
+                    <form style={{ display: "flex", flexDirection: "column" }}>
                         <input
                             type="text"
                             placeholder="Lång jävla länk"
                             value={text}
                             onChange={e => setText(e.target.value)}
                             autoFocus
                         />
-                        <button onClick={submit}>Förkorta</button>
+                        <div>
+                            <label htmlFor="expire-time">Utgångsdatum</label>
+                            <input id="expire-time" type="datetime-local" value={expire} onChange={e => setExpire(e.target.value)} />
+                            <button onClick={e => { e.preventDefault(); setExpire(""); }}>Rensa utgångsdatum</button>
+                        </div>
+                        <div style={{ margin: "10px 0"}}>
+                            <button onClick={submit}>Förkorta</button>
+                        </div>
                     </form>
                     <h2>Specificera förkortad länk</h2>
                     <p>Önska en förkortad länk, exempelvis "ior". Giltiga tecken: A-Ö, a-ö, 0-9</p>
                     <p>Används för exempelvis rekryteringsformulär för nämnder. Du måste vara funktionär för att nyttja denna funktionalitet.</p>
-                    <form className="row">
+                    <form style={{ display: "flex", flexDirection: "column" }}>
```

Comment:
Why are we doing inline style now?

Reply:
I will rewrite the frontend anyways, I'm just lazy atm :D